ngraph/core/builder/CMakeLists.txt

```diff
@@ -37,13 +37,13 @@ if(COMMAND ie_faster_build)
     )
 endif()
 
+target_compile_definitions(${TARGET_NAME} PRIVATE ngraph_EXPORTS)
```

Comment:
Why do we need add definition `ngraph_EXPORTS` for these static libraries? If I remember right we don't use any macro inside the library which depend on `ngraph_EXPORTS`.

Reply:
When we compile `ngraph_builder` against ngraph header, `ngraph_builder` thinks that we import ngraph symbols, but we don't do it. 

As it turned out `ngraph_EXPORTS` does not work as well, because `ngraph_builder` thinks we export ngraph symbols and we have multiple symbols definitions (in `ngraph.lib` and `ngraph_builder.lib`). 

Need to think farther..